x-pack/plugins/fleet/server/services/fleet_usage_sender.ts

```diff
@@ -47,7 +40,7 @@ export class FleetUsageSender {
               try {
                 const usageData = await fetchUsage();
                 appContextService.getLogger().debug(JSON.stringify(usageData));
-                core.analytics.reportEvent('Fleet Usage', usageData);
+                core.analytics.reportEvent('fleet_usage', usageData);
```

Comment:
mind that you need to update the registration name as well in line 80 :)

Reply:
right, thanks